src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/FakeContentType.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Formatting
+{
+    internal class FakeContentType : IContentType
```

Comment:
@olegtk @KirillOsenkov This should be easier 😢

Reply:
➡️ Switched to using **Microsoft.VisualStudio.Text.Implementation** in combination with the `TestComposition` helper from roslyn